sonar-db/src/main/java/org/sonar/db/permission/GroupPermissionDao.java

```diff
@@ -51,16 +51,26 @@ public int countGroups(DbSession session, String permission, @Nullable Long comp
   }
 
   /**
+   * Returns the names of the groups that match the given query, for the given organization.
+   * The virtual group "Anyone" may be returned as the value "zero" (0L).
```

Comment:
this method returns strings. Group anyone is represented by the string "zero" or "0"?